From 0d6f7bcea18c2e6cb6a1bfc33b480253f41de2cc Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jozef=20=C4=8C=C3=AD=C5=BE?= <jozef.ciz.ml@gmail.com>
Date: Tue, 19 Apr 2022 15:52:17 +0200
Subject: [PATCH] Increase width of line number panel

It's common for files to have more than 1000 lines. If the line number
column is not dynamic, then having fixed 4 digits seems like a better
default.
---
 include/zep/editor.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/zep/editor.h b/include/zep/editor.h
index 307d223d..dc4030a9 100644
--- a/include/zep/editor.h
+++ b/include/zep/editor.h
@@ -189,7 +189,7 @@ const float bottomBorder = 2.0f;
 const float textBorder = 2.0f;
 const float tabToneLine = 5.0f;
 const float tabSpacing = 1.0f;
-const float leftBorderChars = 3;
+const float leftBorderChars = 4;
 
 #define DPI_VEC2(value) (value * GetEditor().GetDisplay().GetPixelScale())
 #define DPI_Y(value) (GetEditor().GetDisplay().GetPixelScale().y * value)
-- 
2.39.1

